drivers/NRF5/Radio_ESB.h

```diff
@@ -53,7 +53,7 @@
 #define NRF5_ESB_ARC (15)
 
 // How often broadcast messages are send
-#define NRF5_ESB_BC (4)
+#define NRF5_ESB_BC (1)
```

Comment:
if set to (1) BC msgs won't be sent (due to [this](https://github.com/mysensors/MySensors/pull/830/files/9e9a03f8aeb286e2a21592fff9eb9a84801939a7..60f04e4edd717d8c9d98db04620efc8c2f2883ef#diff-d9fc3fc33e5f695ae5f76dfa3d11fc43L404))